### DISCUSSION TITLE
craftable medicine

### DISCUSSION THREAD

**Discussion by robotic-gamer** [CleverRaven/Cataclysm-DDA#48540] [Ideas]
just some general musings on various medicines that could in theory be crafted by a survivor.
let me know if you would like more info on how to craft the more complex ones.
also let me know if all the topic I have been making is too spammy.
lastly does anyone else have other ideas I missed?

- painkillers
	> aspirin
	> wllow bark tea/extract
	> coal tar (contains paracetamol)
	> mint oil (wild herbs)
	> capsaicin (chili powder)
	> weed
	> alcohol
	> poppy painkillers
	> raw poppy (inferior poppy painkiller)
	> pethidine (opioid)
	charcoal + phenol +(zinc+ aluminum)+ hydrochloric acid+ coal+ ammonia+ lye 
        powder + methanol + ethanol + sulfur + bleach + sulfuric acid + lye
	--silver--->pethidine
	
- disinfectant
	> alcohol
	> vinegar
	> bleach
	> phenol
	> turpentine

- antibiotics
	> sulfa drugs (narrow spectrum antibiotics)
	ammonia +phenol+acetic anhydride+hydrocholoric acid+sulfuric acid+lye-->sulfa drug
	> phage therapy (culture infections bacteria, then isolate viruses then inject)
	> mold extract (impure antibiotics)

- stimulants
	> smelling salts (ammonia with some chemistry---->ammonium carbonate)
	> amphetamine
	phenol+(zinc or aluminum)+glycerol + formic acid + ammonia ->amphetamine
	> datura seeds
	> adrenal gland extract
	> nicotine

- recreational substances
	> alcohol
	> weed
	> solvents (gasoline, markers, acetone etc.)
	> hallucinogens (mushroom extract)

- sleeping aids
	> alcohol
	> Paraldehyde
	ethanol----silver,sulfuric acid------>Paraldehyde
	> weed

- mood stabilizer
	> lithium salts
	burn certain batteries to ash, 
	add acetic acid, 
	concentrate (boil down),
	achieve known contraction through limited water
	isolate salt into pills
	
- anti depressants
	> st. john wort (common weed in new england)

- anti fungals
	> conversion of athletes foot cream to oral formulation
	> main issue here is that the vast majority of antifungals are tropical, so changing fungal infections to be treated tropical first 
          before it gets under the skin.
	> similar to bites treated with antiseptics before it becomes an infection
	> this would allow more realistic anti-fungals

- supplements
   > vitamin c ( fruit converted to protein powder like powder)
   > calcium (limestone or bone meal tablets)
   > iron (scrap metal dissolved in acid to make iron salts)
   > b12 (protein powder)

 - anti-nausea
	> weed
	> datura seed

 - anti-radiation
	> iodine
	> Prussian blue
	> diuretics (dandelion, alcohol, coffee,etc)
	> oxidizer powder (perchlorate ions, mimics iodine)

-  anti-food posioning
	> charcoal

- cold medicine
	> mint oil
	> bee balm
	> camphor
	turpentine +acetic acid +strong acid + water + oxidizer powder-->camphor

- misc chemistry for completeness
	> turpentine from pine bough
	> phenol from willow bark
	> sulfuric acid from gypsum
	> sulfur from sulfuric acid and aluminum powder
	> lye from lye powder
	> acetic anhydrate from acetone and acetic acid
	> acetone from limestone and [vinegar/acetic acid]
        > ammonia via distilling large amounts of bird litter
        > ammonia via Calcium cyanamide
        CaC2+N2--heat-->CaCN2 +C
        CaCN2+H2O---->NH3 +CaCO3

edit :added ammonia production methods

**Comment by faefux** [comment]
>poppy painkillers
>weed
I think the most realistic here would be an alcohol tincture. I don't think I have the files anymore but I already tried this. The big problem is that you use very small doses in a tincture so you end up getting huge amounts of painkillers fairly easily and it becomes a balance issue as all ingredients are very easily accessible that a single bottle of rum and handful of poppy pods will essentially gives you a play-through worth of painkiller.

I'd be happy to attempt to implement it again if you have an interesting alternative or any thoughts.

**Reply by actual-nh** [reply]
I may need to double-check on whether it makes it more bioavailable, but I doubt an alcoholic extraction of poppies - laudanum - would be significantly more potent as a painkiller than you could get from the same number of poppies. Laudanum was a fairly mild painkiller. It was more notable as a sedative (very useful if you already wanted/needed to sleep and as a way to make a death more peaceful) and as - IF it was made accurately AND hadn't evaporated much; the latter happening killed some people with opioid overdose - a way to get smaller doses than would be practical with more-direct use.

**Comment by actual-nh** [comment]
Quite a few of these do have recipes in the game, BTW. Alternative methods/materials for these are still of interest, of course.

Regarding anti-radiation, I agree that more (and more realistic) anti-radiation measures would be helpful.

> diuretics (dandelion, alcohol, coffee,etc)

What's the logic here? Vs tritium, or simply trying to remove more water-soluble radioactive materials (not sure how well the latter would work - will have to look it up)?

> oxidizer powder (perchlorate ions, mimics iodine)

I rather doubt that; the thyroid is quite selective about using iodine (people get goiter by it swelling if it's having problems gathering enough). Evidence?

**Reply by robotic-gamer** [reply]
I figured diuretics plus multivitamin would probably have some small effect at getting rid of radioactive material

perchlorate is used to reduce iodine uptake in the thyroid gland. the point of taking iodine for radiation poisoning is to reduce radioactive iodine uptake in the thyroid gland. in principle perchlorate can thus reduce radioactive iodine uptake.
